language/snippets/src/main/java/com/example/language/Analyze.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google Inc.
+ * Copyright 2023 Google Inc.
```

Comment:
Please, rollback copyright year. We setup the current year only for new files.

Reply:
Done.